data/Mayank-Sharma17.json

```
@@ -0,0 +1,42 @@
+{
+  "name": "Mayank Sharma",
+  "type": "personal",
+  "bio": "Open Source Contributor **@Hacktoberfest** | java | DSA | frontend development",
+  
+  "tags": [
+    "Open Source",
+    "GitHub",
+    "Gitbash",
+    "Java",
+    "DSA",
+    "Frontend Developer",
+    "Bootstrap",
+    "jQuery"
+  ],
+  "socials": [
+    { "icon": "FaLinkedin", "url": "https://linkedin.com/in/mayank257" },
+    { "icon": "FaGithub", "url": "https://github.com/Mayank-Sharma17" },
+    { "icon": "FaTwitter", "url": "https://twitter.com/@MayankS93339625" }
+  ],
+  "links": [
+    {
+      "group": "Socials Primary",
+      "name": "Twitter: Follow me",
+      "url": "https://twitter.com/@MayankS93339625",
```

Please fix this broken URL. Rest looks good to me. 

```suggestion
      "url": "https://twitter.com/MayankS93339625",
```

---

oops, thank you @kumarsonsoff3 for telling